src/script/app.js

```diff
@@ -0,0 +1,38 @@
+// Hidden navbar
+let lastKnownScrollPosition = 0;
+
+document.addEventListener("scroll", function () {
+  if (window.scrollY > lastKnownScrollPosition) {
+    document.querySelector(".nav").classList.add("nav_hidden");
+  } else {
+    document.querySelector(".nav").classList.remove("nav_hidden");
+  }
+  lastKnownScrollPosition = window.scrollY;
+});
+
+// Newsletter
+const text = document.querySelector(".text");
+const email = document.querySelector(".form_email");
+const button = document.querySelector(".form_button");
+let regExp = /^[^ ]+@[^ ]+\.[a-z]{2,3}$/;
+
+email.addEventListener("input", function () {
+  if (email.value.match(regExp)) {
+    text.innerHTML = "Your address email in valid";
+    text.classList.add("text_true");
+    text.classList.remove("text_false");
```

Comment:
The class should have a clear meaning, in this case it does not, consider changing it to `text_valid` & `text_invalid`.